src/js/tech/html5.js

```diff
@@ -70,6 +72,13 @@ class Html5 extends Tech {
       }
     }
 
+    if (this.featuresNativeVideoTracks) {
+      this.handleVideoTrackChange_ = Fn.bind(this, this.handleVideoTrackChange);
```

Comment:
I don't like the fact we don't remember *why* we have to do this weird dance here. @BrandonOCasey would you mind giving this a shot without the binding thing in Safari?